cmd/tsconnect/package.json

```diff
@@ -13,7 +14,8 @@
     "xterm-addon-fit": "^0.5.0"
   },
   "scripts": {
-    "lint": "tsc --noEmit"
+    "lint": "tsc --noEmit",
+    "pkg-types": "dts-bundle-generator --inline-declare-global=true --no-banner -o pkg/pkg.d.ts src/pkg/pkg.ts"
```

Comment:
Eventually we'll likely want to split some of the IPN types into its own package. For example, [in the Docker Extension](https://github.com/tailscale/docker-extension/blob/main/ui/src/tailscale.ts#L8) I needed to manually re-implement all the IPN types too.

Nothing to do now, just flagging FYSA.

Reply:
Huh, hadn't realized that the Docker extension has a TypeScript component, that's a good observation.